support-workers/deployToCode.mjs

```diff
@@ -0,0 +1,81 @@
+import { S3Client, PutObjectCommand } from '@aws-sdk/client-s3';
+import {
+	LambdaClient,
+	UpdateFunctionCodeCommand,
+} from '@aws-sdk/client-lambda';
+import { createReadStream } from 'fs';
+import { fileURLToPath } from 'url';
+import { dirname, join } from 'path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const S3_BUCKET = 'membership-dist';
```

Comment:
Maybe not one for now but in future we could read this from parameter store: `/account/services/artifact.bucket`